test/runtests.jl

```diff
@@ -116,6 +116,17 @@ Si        13.00000000      14.00000000      $(frame+1).00000000          0
             @test all(seq1 .≈ seq2)
         end
 
+        @testset "missingfile" begin
+            try
+                read_frame("bla.extxyz")
+            catch e
+                buf = IOBuffer()
+                showerror(buf, e)
+                message = String(take!(buf))
+                @test message == "file bla.extxyz cannot be opened for reading"
```

Comment:
Why not use an @test_throws or @test_logs

Reply:
Will change this, thanks. I found some comments on stack overflew about @test_throws not being that general.